assignments/play_with_array/play_with_array.js

```diff
@@ -21,6 +39,10 @@ function multiplyByN(arr, n) {
 }
 
 function removeNthElement(arr, n) {
+
+
+    arr.splice(n,1)
+    return arr
```

Comment:
@Rajan99sharma99 

Hai Rajan Sharma 

I Like the way you use the slice in this function. Looks good.


Thank You.